flow/settings.ini

```diff
@@ -8,6 +8,9 @@ sonar_runner = #TODO add location to sonar runner
 [tracker]
 url = https://www.pivotaltracker.com
 
+[jira]
+url = https://thd.atlassian.net
```

Comment:
> I wonder if there is a more generic jira URL we can use as the default instead of one specific to our workspace?

That URL is our highest level URL for THD Jira.  Otherwise you just get routed to the atlassian private nets.  If you want the Atlassian public page it is www.atlassian.com.  Is there a reason you don't want to use our entry point for the default?

Reply:
Per https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/
```
The URIs for resources have this structure:

https://<site-url>/rest/api/3/<resource-name>
```
So I think whomever is using it will have to insert a workspace specific url.  Maybe we change it to something generic here and the internal fork uses our specific url?

Reply:
If I am reading this correctly the <site-url> will always be https://thd.atlassian.net for all of our API calls unless someone is running something against our sandbox instance which has a different base url and really should only should be used for testing.

Reply:
Based on the Slack message from Jeff then for the Open Source we should use: www.atlassian.net

Reply:
> Per https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/
> 
> ```
> The URIs for resources have this structure:
> 
> https://<site-url>/rest/api/3/<resource-name>
> ```
> 
> So I think whomever is using it will have to insert a workspace specific url. Maybe we change it to something generic here and the internal fork uses our specific url?

Yes - that is the correct approach

Reply:
updated to use www.atlassion.net